src/Uno.UWP/Devices/Bluetooth/BluetoothDevice.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Devices.Bluetooth
+{
+	public partial class BluetoothDevice : global::System.IDisposable
+	{
+		private static string _deviceSelectorPrefix = "System.Devices.DevObjectType:=5 AND System.Devices.Aep.ProtocolId:=\"{E0CBF06C-CD8B-4647-BB8A-263B43F0F974}\" AND ";
+		private static string _deviceSelectorIssueInquiry = "System.Devices.Aep.Bluetooth.IssueInquiry:=System.StructuredQueryType.Boolean";
+
+		public static string GetDeviceSelector()
+		{
+			return _deviceSelectorPrefix + "(System.Devices.Aep.IsPaired:=System.StructuredQueryType.Boolean#True OR " + _deviceSelectorIssueInquiry + "#False)";
+		}
+
+		public static string GetDeviceSelectorFromPairingState( bool pairingState)
+		{
+			if(pairingState )
+			{
+				return _deviceSelectorPrefix + "(System.Devices.Aep.IsPaired:=System.StructuredQueryType.Boolean#True OR " + _deviceSelectorIssueInquiry + "#False)";
+			}
+			else
+			{
+				return _deviceSelectorPrefix + "(System.Devices.Aep.IsPaired:=System.StructuredQueryType.Boolean#False OR " + _deviceSelectorIssueInquiry + "#True)";
+			}
+		}
+
+		public static string GetDeviceSelectorFromConnectionStatus( BluetoothConnectionStatus connectionStatus)
+		{
+			if(connectionStatus == BluetoothConnectionStatus.Connected )
+			{
+				return _deviceSelectorPrefix + "(System.Devices.Aep.IsConnected:=System.StructuredQueryType.Boolean#True OR " + _deviceSelectorIssueInquiry + "#False)";
+			}
+			else
+			{
+				return _deviceSelectorPrefix + "(System.Devices.Aep.IsConnected:=System.StructuredQueryType.Boolean#False OR " + _deviceSelectorIssueInquiry + "#True)";
+			}
+		}
+
+		public static string GetDeviceSelectorFromDeviceName( string deviceName)
+		{
+			return _deviceSelectorPrefix + "(System.ItemNameDisplay:=\"" + deviceName + "\" OR " + _deviceSelectorIssueInquiry + "#True)";
+		}
+
+		public static string GetDeviceSelectorFromBluetoothAddress( ulong bluetoothAddress)
+		{ 
+			string macAddr = string.Format("{0:x12}", bluetoothAddress);
+			return _deviceSelectorPrefix + "(System.DeviceInterface.Bluetooth.DeviceAddress:=\"" + macAddr + "\" OR " + _deviceSelectorIssueInquiry + "#True)";
+		}
+
+		//method public static string GetDeviceSelectorFromClassOfDevice( BluetoothClassOfDevice classOfDevice)

```

Comment:
Please remove

Reply:
It takes some time to dig into working of this method; if someone wants to implement this method, he/she can use this info and save about hour... Really should I delete this?

Reply:
Can I leave this comment in code?

Reply:
Can I leave this comment in code? if someone wants to implement this method, he/she can use this info and save some time (results of checks how this method work on UWP is described).

Reply:
Can I get answer to this question?

Reply:
Yes, please remove this. If this is useful information, please add this to a documentation file in the `features` directory

Reply:
Ok, I will move this info (how to implement this method) to new `features` page.